src/Modal/index.jsx

```diff
@@ -20,12 +22,30 @@ class Modal extends React.Component {
 
     this.headerId = newId();
 
+    const modalRootId = 'modal-root';
+    this.modalRoot = document.getElementById(modalRootId);
```

Comment:
What happens if there are two modals on the same page if `id`s are supposed to be unique across the DOM?